RedHat-UX/red-hat-design-system
# docs(tooltip): update best practices

## What I did

1. Updated "Best practice" section for tooltip to show do/don't examples


## Testing Instructions

1. Check `<rh-tooltip>`'s ["Best practices" section](https://deploy-preview-2030--red-hat-design-system.netlify.app/elements/tooltip/guidelines/#best-practices) in the DP.

## Notes to Reviewers
